packages/wallet/src/contract-tests/transactions.test.ts

```diff
@@ -47,9 +47,11 @@ describe('transactions', () => {
   }
 
   async function testTransactionSender(transactionToSend) {
-    const saga = transactionSender(transactionToSend, 'channelId');
+    const process = 'DirectFunding';
```

Comment:
I dislike the word `process`, because you are overwriting a global variable here.

Reply:
Renamed to procedure